api/src/main/java/io/smartcat/spring/cassandra/showcase/adapter/persistence/cassandra/account/AccountRepositoryCassandra.java

```diff
@@ -53,7 +53,8 @@ public void setupTable() {
 
     @Override
     public Optional<Account> accountOfEmail(final EmailAddress email) {
-        final Statement statement = accountByEmailMapper.getQuery(email.address());
+        final Statement statement = accountByEmailAccessor
+            .getAccountByEmailAddress(email.address());
```

Comment:
@mgobec Why test failed here? It returned accountByEmail with all null values and when I changed to accessor it works perfectly?